integration/angular_cli/e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../out-tsc/e2e",
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2018",
```

Comment:
Have you verified that the oldest version of node that we support is compatible with full es2018? I think it might be but I'm not sure.

Reply:
The oldest Node version that we support is `10.13.x`, it seems that the only thing which is not supported is https://tc39.es/ecma262/#sec-proxy-object-internal-methods-and-internal-slots-ownpropertykeys

See: https://node.green/#ES2018